### DISCUSSION TITLE
Who will do the dynamic effect of Pixi JS official website?

### DISCUSSION THREAD

**Discussion by leonardo110** [pixijs/pixijs#9647] [Q&A] [answered]
I really want to know how the particle effect of the official website is achieved, looking at the effect thief cool, does anyone know, please tell me, thank you

**Comment by SuperSodaSea** [comment, marked as answer]
Unfortunately, It seems that the original code for that animation was lost. You can find the obfuscated code at <https://github.com/pixijs/pixijs.com/blob/2d7b9ecd05031571cfaf971eedc5a4c021e6553e/static/header/js/nebulon.js>.

**Reply by leonardo110** [reply]
Thank you very much. That's done. Thank you very much ！！！！！！！！！！！！！！！！

**Comment by leonardo110** [comment]
@jupiter0913 不知道能不能艾特到你，你可以看看的我的网站效果  https://quancy.cn
